setup.js

```diff
@@ -0,0 +1,6 @@
+import 'dotenv/config';
+import { User } from './src/models/user.js';
+import { Token } from "./src/models/token.js";
+import { client } from './src/utils/db.js';
+
+client.sync({ force: true });
```

Comment:
Using `force: true` in `client.sync()` will drop all existing tables and recreate them. This is useful for development but can lead to data loss in production. Consider using `alter: true` for a safer synchronization that only alters tables to match the model.